lib/charliecloud.py

```diff
@@ -613,20 +639,32 @@ def exit(code):
 
 def init(cli):
    # logging
-   global log_festoon, log_fp, trace_fatal, verbose, save_xattrs
-   assert (0 <= cli.verbose <= 3)
-   verbose = cli.verbose
+   global log_festoon, log_fp, log_level, trace_fatal, save_xattrs
    save_xattrs = (not cli.no_xattrs)
    trace_fatal = (cli.debug or bool(os.environ.get("CH_IMAGE_DEBUG", False)))
+   if (cli.quiet):
+      fail_ct = 0
+      if (trace_fatal):
+         ERROR("“quiet” incompatible with “debug” and “CH_IMAGE_DEBUG”")
+         fail_ct += 1
+      if (cli.verbose):
+         ERROR("“quiet” incompatible with “verbose”")
+         fail_ct += 1
```

Comment:
We could do it where every `-v` increases verbosity by 1 and every `-q` decreases it by 1. I wonder if that's too weird though.